sunnyportal/responses.py

```diff
@@ -43,7 +43,7 @@ def __str__(self):
 
 
 Yield = namedtuple('Yield', ['timestamp', 'absolute', 'difference'])
-Power = namedtuple('Power', ['timestamp', 'power'])
+Power = namedtuple('Power', ['timestamp', 'min_power','mean_power','max_power'])
```

Comment:
I've merged these changes in a slightly different form (see 45bcd9d6). The `power` member is still there and the other two are called `min` and `max`.

Please rebase your changes ontop of this.